eppoclient/client_test.go

```diff
@@ -290,3 +291,39 @@ func Test_client_correctActionIsReturnedIfBanditLoggerPanics(t *testing.T) {
 		Action:    &expectedAction,
 	}, result)
 }
+
+func Test_Initialized_timeout(t *testing.T) {
+	var mockLogger = new(mockLogger)
+	client := newEppoClient(newConfigurationStore(), nil, nil, mockLogger, applicationLogger)
+
+	timedOut := false
+	select {
+	case <-client.Initialized():
+		timedOut = false
+	case <-time.After(1 * time.Millisecond):
+		timedOut = true
+	}
+
+	assert.True(t, timedOut)
+}
+
+func Test_Initialized_success(t *testing.T) {
+	var mockLogger = new(mockLogger)
+	configurationStore := newConfigurationStore()
+	client := newEppoClient(configurationStore, nil, nil, mockLogger, applicationLogger)
+
+	go func() {
+		<-time.After(1 * time.Microsecond)
+		configurationStore.setConfiguration(configuration{})
+	}()
+
+	timedOut := false
+	select {
+	case <-client.Initialized():
+		timedOut = false
+	case <-time.After(1 * time.Millisecond):
+		timedOut = true
+	}
+
+	assert.False(t, timedOut)
```

Comment:
Big picture question to remind myself how go channels work - assuming the customer is running an HTTP server and has evaluations they want to perform in a variety of handlers with their goal being NOT to perform evaluations until the SDK is initialized, would they add a reference to this channel to a `Context` they pass around and read the channel, or keep a reference to a global eppo SDK and call `client.Initialized`? Do you have a small code example to share?